Dockerfile

```diff
@@ -33,7 +33,11 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
     libgbm1:i386 \
     libcurl4:i386 \
     libfuse2 \
-  && curl https://chromium.googlesource.com/chromium/src/+/HEAD/build/install-build-deps.sh\?format\=TEXT | base64 --decode | cat > /setup/install-build-deps.sh \
+    software-properties-common
+
+  RUN add-apt-repository ppa:git-core/ppa -y && apt-get update
+
+  RUN curl https://chromium.googlesource.com/chromium/src/+/HEAD/build/install-build-deps.sh\?format\=TEXT | base64 --decode | cat > /setup/install-build-deps.sh \
```

Comment:
This should all be chained into the one `RUN` command to reduce docker image size

Reply:
TIL - Ok gotcha will do.